src/main/menus/app.js

```diff
@@ -224,6 +224,13 @@ var createTemplate = function(mainWindow, config) {
   template.push({
     label: '&Help',
     submenu: [{
+      label: `${app_name} Docs`,
+      click: function() {
+        electron.shell.openExternal('http://docs.mattermost.com')
```

Comment:
Instead of docs.mattermost.com, propose linking to the Desktop documentation at http://docs.mattermost.com/help/getting-started/setup.html

In general, that page needs to be re-organized to be about Desktop Apps in general, their functionality, and a link to their install procedures--right now it's like an install procedure with help tacked on...